Content.Shared/Silicons/Laws/Components/StartIonStormedComponent.cs

```diff
@@ -0,0 +1,15 @@
+
+namespace Content.Shared.Silicons.Laws.Components;
+
+/// <summary>
+/// Applies law altering ion storms on a specific entity IonStormAmount times when the entity is spawned.
+/// </summary>
+[RegisterComponent]
+public sealed partial class StartIonStormedComponent : Component
+{
+    /// <summary>
+    /// Amount of times that the ion storm will be run on the entity on spawn.
+    /// </summary>
+    [DataField]
+    public int IonStormAmount = 1;
+}
```

Comment:
Are you using an IDE or something like notepad? Our .editorconfig is set up to add a new line at the end of each file, so I wonder why there is one missing here.

Reply:
For .cs i used Visual Studio 2022.